src/dev/build/tasks/copy_source_task.js

```diff
@@ -47,6 +47,7 @@ export const CopySourceTask = {
         'webpackShims/**',
         'config/kibana.yml',
         'tsconfig*.json',
+        '.i18nrc.json',
```

Comment:
This is necessary because we're now using `.i18nrc.json` files at runtime to load translation files, is that correct?

Reply:
Yes. Instead of traversing all the source code for looking for transaltion file patterns, we look into the `.i18nrc.json` file now. There are two i18nrc files, one at root and the other in the `x-pack` dir.